### PR TITLE
Add copy shortcuts ctrl+c ctrl+v to diagram

### PR DESCRIPTION
**Pull request by mgonc**

### CODE REVIEW

**Comment by mgonc** [comment]
#28

File: src/components/Diagram.vue [hunk 1]
```diff
@@ -733,6 +736,7 @@
                     let elem = document.getElementById(self.selectedTask.id);
                     let inc = ev.ctrlKey ? 10 : 1;
                     let v = 0;
+                    debugger;
```
Comment (waltersf) [inline src/components/Diagram.vue]: You left a debugger statement. I'll remove it.

**Comment by mgonc** [comment]
removed

**Comment by waltersf** [comment]
@mgonc The changes in scrolling are causing an error in the scoll bar in the diagram and made it not accept scrolling with mouse wheel:

vue.runtime.esm.js:619 [Vue warn]: Error in mounted hook: "TypeError: i[e] is not a function"

```
---> <VuePerfectScrollbar>
       <Diagram> at src/components/Diagram.vue
         <BTab>
           <BTabs>
             <Anonymous>
               <App> at src/App.vue
                 <Root>
warn @ vue.runtime.esm.js:619
logError @ vue.runtime.esm.js:1884
globalHandleError @ vue.runtime.esm.js:1879
handleError @ vue.runtime.esm.js:1839
invokeWithErrorHandling @ vue.runtime.esm.js:1862
callHook @ vue.runtime.esm.js:4213
insert @ vue.runtime.esm.js:3139
invokeInsertHook @ vue.runtime.esm.js:6340
patch @ vue.runtime.esm.js:6559
Vue._update @ vue.runtime.esm.js:3942
updateComponent @ vue.runtime.esm.js:4060
get @ vue.runtime.esm.js:4473
run @ vue.runtime.esm.js:4548
flushSchedulerQueue @ vue.runtime.esm.js:4304
(anonymous) @ vue.runtime.esm.js:1980
flushCallbacks @ vue.runtime.esm.js:1906
Promise.then (async)
timerFunc @ vue.runtime.esm.js:1933
nextTick @ vue.runtime.esm.js:1990
queueWatcher @ vue.runtime.esm.js:4396
update @ vue.runtime.esm.js:4538
notify @ vue.runtime.esm.js:730
reactiveSetter @ vue.runtime.esm.js:1055
proxySetter @ vue.runtime.esm.js:4625
(anonymous) @ WorkflowEdit.vue:539
Promise.then (async)
(anonymous) @ WorkflowEdit.vue:537
Promise.then (async)
load @ WorkflowEdit.vue:528
mounted @ WorkflowEdit.vue:459
invokeWithErrorHandling @ vue.runtime.esm.js:1854
callHook @ vue.runtime.esm.js:4213
insert @ vue.runtime.esm.js:3139
invokeInsertHook @ vue.runtime.esm.js:6340
patch @ vue.runtime.esm.js:6559
Vue._update @ vue.runtime.esm.js:3942
updateComponent @ vue.runtime.esm.js:4060
get @ vue.runtime.esm.js:4473
run @ vue.runtime.esm.js:4548
flushSchedulerQueue @ vue.runtime.esm.js:4304
(anonymous) @ vue.runtime.esm.js:1980
flushCallbacks @ vue.runtime.esm.js:1906
Promise.then (async)
timerFunc @ vue.runtime.esm.js:1933
nextTick @ vue.runtime.esm.js:1990
queueWatcher @ vue.runtime.esm.js:4396
update @ vue.runtime.esm.js:4538
Vue.$forceUpdate @ vue.runtime.esm.js:3963
(anonymous) @ index.js:178
(anonymous) @ index.js:154
(anonymous) @ index.js:119
(anonymous) @ App.vue:32
(anonymous) @ App.vue:36
hotApply @ bootstrap:660
(anonymous) @ bootstrap:342
Promise.then (async)
hotUpdateDownloaded @ bootstrap:341
hotAddUpdateChunk @ bootstrap:317
webpackHotUpdateCallback @ bootstrap:36
(anonymous) @ app.40bae04bb2ea03801489.hot-update.js:1
vue.runtime.esm.js:1888 TypeError: i[e] is not a function
    at index.js:1
    at Array.forEach (<anonymous>)
    at Object.t.exports [as initialize] (index.js:1)
    at VueComponent.__init (index.js:1)
    at VueComponent.mounted (index.js:1)
    at invokeWithErrorHandling (vue.runtime.esm.js:1854)
    at callHook (vue.runtime.esm.js:4213)
    at Object.insert (vue.runtime.esm.js:3139)
    at invokeInsertHook (vue.runtime.esm.js:6340)
    at VueComponent.patch [as __patch__] (vue.runtime.esm.js:6559)
```

**Comment by mgonc** [comment]
@waltersf fixed

**Pull request merged**